lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
using binary.find() from node-pre-gyp does exactly that, it loads the package.json, and resolves the name and path to xmljs.node from the binary.module_name and binary.module_path defined in package.json

no need to even specify the name `xmljs` or `xmljs.node` because its already defined in package.json

I'm working on a PR for the bindings module to add the node-pre-gyp path like this, no idea if that'll go through